Makefile

```
@@ -36,4 +36,27 @@ validate-exception-sorting:
 	done; \
 	if [ "$$PRINT_ERR" = 0 ]; then \
 		echo "Run the following command to sort all files recursively: make sort-exception-files"; \
-	fi; \
\ No newline at end of file
+	fi; \
+
+dedup-exception-files:
+	@echo "Deduplicating exception files..."
+	@find $(EXCEPTIONS_DIR) -type f -name '$(EXCEPTIONS_FILE_PATTERN)' -exec sh -c ' \
```

```suggestion
	@find $(EXCEPTIONS_DIR) -type f -name '$(EXCEPTIONS_FILE_PATTERN)' -exec sh -c ' \
	@find $(EXCEPTIONS_DIR) -type f -name '$(EXCEPTIONS_FILE_PATTERN)'  | xargs -P 4 -I {} sh -c '
    if ! jq '\''keys[0] as $$key | { ($$key): (.[$$key] | group_by(.Namespace, .ResourceName) | map(.[0])) }'\'' "$$1" > "$$1.tmp"; then
```
This will allow running in parallel(need to test this in CI)

---

I'll try it out, it could also be added to `sort-exception-files`.

---

Adding parallelity with 4 processes decreased round-time by ~50%.

<details>
<summary>Test Results</summary>

**Before change:**
```console
$ time make dedup-exception-files
Deduplicating exception files...

real    0m0.692s
user    0m0.423s
sys     0m0.045s
$ time make dedup-exception-files
Deduplicating exception files...

real    0m0.609s
user    0m0.367s
sys     0m0.028s

$ time make dedup-exception-files
Deduplicating exception files...

real    0m0.712s
user    0m0.485s
sys     0m0.024s
```

**After change:**
```console
$ time make dedup-exception-files
Deduplicating exception files...

real    0m0.277s
user    0m0.503s
sys     0m0.105s
$ time make dedup-exception-files
Deduplicating exception files...

real    0m0.276s
user    0m0.521s
sys     0m0.076s
$ time make dedup-exception-files
Deduplicating exception files...

real    0m0.290s
user    0m0.499s
sys     0m0.083s
```
</details>

---

Nice!